FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
```

Comment:
* In Compound, liquidating allows claiming up to 50% (aka "liquidation rate") of the collateral of a position at a 8% discount (was 5% discount pre-Black Thursday). 
* In Aave it's a bit more nuanced https://docs.aave.com/risk/asset-risk/risk-parameters

How do you reach the conclusion about them being more beneficial to liquidators than borrowers?

Reply:
The fact that liquidators can liquidate more than necessary to render a debt "healthy" appears disadvantageous to the borrower. We haven't yet drawn a conclusion, but have a hypothesis which we plan to analyze.

Reply:
I think this conclusion is probably a little more nuanced because of liquidator sizing constraints and system-wide liquidity constraints. For instance, the amount of 'chunking' you can do for a partial liquidation is related (in a quite non-linear way) to asset volatility. If the chunks are too big and you have an asset where there is low liquidity (e.g. Dai in March), there isn't an equilibrium where liquidators 'beat' borrowers